src/style.rs

```diff
@@ -71,14 +71,15 @@
 use std::fmt;
 
 use bitflags::bitflags;
-
-mod color;
-mod stylize;
-
 pub use color::{Color, ParseColorError};
 pub use stylize::{Styled, Stylize};
 pub mod palette;
 
+mod color;
```

Comment:
Move the mod into the mod block out of the use block. (Then use `cargo +nightly fmt` to order them)

```suggestion
pub use stylize::{Styled, Stylize};

pub mod palette;
mod color;
```